crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -676,3 +677,10 @@ fn has_trailing_comma(
         .kind()
         == SimpleTokenKind::Comma
 }
+
+/// Returns `true` if the parameters are parenthesized (as in a function definition), or `false` if
+/// not (as in a lambda).
+pub(crate) fn are_parameters_parenthesized(parameters: &Parameters, locator: &Locator) -> bool {
```

Comment:
Where is this used? Should it be moved closer to the file that is using it?

Reply:
It's used in `placement.rs`. I was torn.